edb/testbase/connection.py

```diff
@@ -144,7 +144,7 @@ async def commit(self) -> None:
     async def _commit(self) -> None:
         query = self._make_commit_query()
         try:
-            await self._connection.execute(query)
```

Comment:
Why this change?

Reply:
Because once the COMMIT fails, the protoocl driver thinks it isn't in a tx anymore and wants to retry it. This prevents that.

There might be a cleaner way but I didn't see it right away.

I'll add a comment.